pandas/core/frame.py

```diff
@@ -6451,11 +6456,38 @@ def value_counts(
         2         2            0.25
         6         0            0.25
         dtype: float64
+
+        **dropna**
+
+        With `dropna` set to `False` we can also count rows with NA values.
+
+        >>> df = pd.DataFrame({'first_name': ['John', 'Anne', 'John', 'Beth'],
+        ...                    'middle_name': ['Smith', pd.NA, pd.NA, 'Louise']})
+        >>> df
+          first_name middle_name
+        0       John       Smith
+        1       Anne        <NA>
+        2       John        <NA>
+        3       Beth      Louise
+
+        >>> df.value_counts()  # dropna is True per default
```

Comment:
Don't think the comment is needed either since the default is specified in the parameters section

Reply:
Removed in f798c5b